src/instructlab/sdg/generate_data.py

```diff
@@ -94,15 +96,15 @@ def _gen_train_data(machine_instruction_data, output_file_train, output_file_mes
                 user += "\n" + synth_example["context"]
             assistant = _unescape(_get_response_hack(synth_example))
             train_entry = {
-                "system": _SYS_PROMPT,
+                "system": system_prompt if system_prompt is not None else _SYS_PROMPT,
```

Comment:
Since we are still using a default system prompt as a fallback, we probably want to update that to be the 8B prompt by default

Reply:
updated, thanks